src/test/java/net/masterthought/cucumber/ConfigurationTest.java

```diff
@@ -140,4 +141,33 @@ public void getProjectName_ReturnsProjectName() {
         // then
         assertThat(name).isEqualTo(projectName);
     }
+    
+    @Test
+	public void getTagsToExcludeFromChart_defaultConfiguration_ReturnsEmptyList() throws Exception {
```

Comment:
1. comments was only for the method name, not the test itself - keep test if you think it is worth!
2. It does not but it increases noise